README.md

```diff
@@ -16,8 +16,17 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
-```
+Clearfix is a means of addressing some issues created when using 'float' for page layout purposes. Clearfix can expand the container the floated element is within, so that the container is intuitively sized around the floated object. Clearfix can be used with the 'before' or 'after' pseudo-classes.
```

Comment:
Minor clarification: A "clearfix" *often* utilizes before/after pseudo classes, so that it is easy to reuse.